src/test/java/org/fenixedu/legalpt/services/commons/export/XmlZipFileWriterTest.java

```diff
@@ -0,0 +1,65 @@
+package org.fenixedu.legalpt.services.commons.export;
+
+import junit.framework.TestCase;
+import net.lingala.zip4j.io.inputstream.ZipInputStream;
+import net.lingala.zip4j.model.LocalFileHeader;
+import org.junit.Test;
+
+import java.io.*;
+
+public class XmlZipFileWriterTest extends TestCase {
+
+    @Test
+    public void testCreateEncryptedZip() throws IOException {
```

Comment:
@joaofreitas-qubit I suggest we had two more tests:

1. Try to use the `java.util.ZipInputStream` which is the regular unzip without password and ensure it fails and cannot read the content of the file
2. Try to provide a wrong password and ensure you cannot get the file content either.